app/controllers/merchant/dashboard_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Merchant::DashboardController < Merchant::BaseController
 
   def show
+    @orders = Order.where(status: :pending).joins(:items).where(items: {merchant_id: current_user.merchant.id}).distinct
   end
```

Comment:
I think this should be abstracted to the model

Reply:
I agree, this prob should be handled and tested in the model/model tests

Reply:
My mentor helped me build this and thought that the logic belonged in the controller. 

Why do you think it should be in the model?